Orange/widgets/utils/state_summary.py

```diff
@@ -101,3 +105,8 @@ def new_line(text):
             details = f'No data on {type_io}.'
         full_details.append(details if not name else f'{name}:<br>{details}')
     return '<hr>'.join(full_details)
+
+
+@summarize.register(Table)
```

Comment:
If we stop supporting Python 3.6, it would suffice to annotate the function's argument as `Table` instead of passing `Table` to decorator.

Reply:
According to NEP 29, we've been free to drop 3.6 since Jun 23, 2020. The only concern would be Anaconda (for the same reason we still tacitly support PyQt5.9).
https://numpy.org/neps/nep-0029-deprecation_policy.html